src/plugins/usage_collection/public/index.ts

```diff
@@ -6,12 +6,12 @@
  * Side Public License, v 1.
  */
 
-import { PluginInitializerContext } from '../../../core/public';
+import type { PluginInitializerContext } from 'src/core/public';
 import { UsageCollectionPlugin } from './plugin';
 
-export { METRIC_TYPE } from '@kbn/analytics';
```

Comment:
Removed as we're using `import { METRIC_TYPE } from '@kbn/analytics'` everywhere except in `lens` (which I changed in this PR).